docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '3'
+
+networks:
+  db-data-net:
+    driver: bridge
+
+services:
+  mysql:
+    image: mysql-local:2.0
+    build:
+      context: .
+      dockerfile: Dockerfile.mysql
+    container_name: mysql-container
+    ports:
+      - "3306:3306"
+    volumes:
+      - db-data:/var/lib/mysql
+    environment:
+      - MYSQL_ROOT_PASSWORD=1234
+      - MYSQL_DATABASE=app_db
+      - MYSQL_USER=app_user
+      - MYSQL_PASSWORD=1234
```

Comment:
These envs are set up on the Dockerfile of MySQL, no need to define it one more time